src/components/useCallback.jsx

```diff
@@ -1,7 +1,46 @@
-export const Callback=()=>{
+import { useState, useCallback } from "react";
 
-    return (
-        <div>Callback</div>
-    );
-    
-    }
\ No newline at end of file
+export const Callback = () => {
+  let [num, setNum] = useState(0);
+
+  let [col, setCol] = useState(false);
+
+  function costly(x) {
+    for (let i = 0; i < 1000000000; i++) {
+      x += i;
+    }
+    return x;
+  }
+
+  function getClick() {
+    setNum(num + 1);
+  }
+
+  const colorChange = {
+    color: col === false ? "red" : "green",
+  };
+
+  //costly(num);
+
+  useCallback(() => {
+    costly(num);
+  }, []); //This command when activated will instantaneously change color as well as update the count as compared to time it was taking to do same in before mentioned command
```

Comment:
you should have everything that is used in useCallback , It should also be present in the dependency array.

Reply:
have created a new program and pushed it into same useCallback branch